calicoctl/calico_ctl/status.py

```diff
@@ -252,3 +273,42 @@ def pprint_bird_protocols(version):
         print str(x) + "\n"
     else:
         print results + "\n"
+
+
+def _gobgp(cmd):
+    if getattr(sys, 'frozen', False):
+        gobgp = sys._MEIPASS + "/gobgp"
+    else:
+        gobgp = "gobgp"
+    results = subprocess32.check_output([gobgp, "-j"] + cmd.split(" "))
+    return json.loads(results)
+
+
+def pprint_gobgp_protocols(version):
+    x = PrettyTable(["Peer address", "Peer type", "State", "Since", "Info"])
+
+    now = int(time.mktime(datetime.datetime.now().timetuple()))
+    try:
+        neighbors = _gobgp("neighbor -t ipv%d" % version)
+    except subprocess32.CalledProcessError:
```

Comment:
Also, the `/var/log/calico/calico-bgp-daemon/current` logs don't show anything that suggest gRPC error:

```bash
time="2016-09-19T22:14:43Z" level=debug msg=sent Key=172.17.8.101 State="BGP_FSM_ESTABLISHED" Topic=Peer data=&{{[] 19 4} 0x13af3e8}
time="2016-09-19T22:14:56Z" level=debug msg="create Destination" Key="192.168.9.128/26" Topic=Table
time="2016-09-19T22:14:56Z" level=debug msg="Processing destination" Key="192.168.9.128/26" Topic=table
time="2016-09-19T22:14:56Z" level=debug msg="computeKnownBestPath known pathlist: 1" Topic=Table
2016-09-19 22:14:56.159692 I | add path: { 192.168.9.128/26 | src: local, nh: 172.17.8.102 }
time="2016-09-19T22:14:56Z" level=debug msg="sent update" Key=172.17.8.101 State="BGP_FSM_ESTABLISHED" Topic=Peer attributes=[{Origin: i}  {Nexthop: 172.17.8.102} {LocalPref: 100}] nlri=[192.168.9.128/26] withdrawals=[]
time="2016-09-19T22:15:13Z" level=debug msg=sent Key=172.17.8.101 State="BGP_FSM_ESTABLISHED" Topic=Peer data=&{{[] 19 4} 0x13af3e8}
time="2016-09-19T22:15:43Z" level=debug msg=sent Key=172.17.8.101 State="BGP_FSM_ESTABLISHED" Topic=Peer data=&{{[] 19 4} 0x13af3e8}
time="2016-09-19T22:16:13Z" level=debug msg=sent Key=172.17.8.101 State="BGP_FSM_ESTABLISHED" Topic=Peer data=&{{[] 19 4} 0x13af3e8}
time="2016-09-19T22:16:43Z" level=debug msg=sent Key=172.17.8.101 State="BGP_FSM_ESTABLISHED" Topic=Peer data=&{{[] 19 4} 0x13af3e8}
time="2016-09-19T22:17:13Z" level=debug msg=sent Key=172.17.8.101 State="BGP_FSM_ESTABLISHED" Topic=Peer data=&{{[] 19 4} 0x13af3e8}
time="2016-09-19T22:17:43Z" level=debug msg=sent Key=172.17.8.101 State="BGP_FSM_ESTABLISHED" Topic=Peer data=&{{[] 19 4} 0x13af3e8}
```